domain/src/main/java/org/oppia/android/domain/spotlight/SpotlightStateController.kt

```diff
@@ -97,6 +101,13 @@ class SpotlightStateController @Inject constructor(
       }
   }
 
+  private fun spotlightStateRetrieverHelper(spotlightViewState: SpotlightViewState):
+    SpotlightViewState {
+      return if (spotlightViewState == SpotlightViewState.SPOTLIGHT_VIEW_STATE_UNSPECIFIED) {
+        SpotlightViewState.SPOTLIGHT_NOT_SEEN
+      } else spotlightViewState
+    }
```

Comment:
```suggestion
  private fun spotlightStateRetrieverHelper(
    spotlightViewState: SpotlightViewState
  ): SpotlightViewState {
    return if (spotlightViewState == SpotlightViewState.SPOTLIGHT_VIEW_STATE_UNSPECIFIED) {
      SpotlightViewState.SPOTLIGHT_NOT_SEEN
    } else spotlightViewState
  }
```

Breaking in this way avoids the extra awkward indentation being needed.

Reply:
done